cw-sns-secretsmanager/README.md

```diff
@@ -0,0 +1,62 @@
+# AWS Service 1 to AWS Service 2
+
+This pattern explains how to deploy a SAM application that Detects and Notify on Amazon Secrets Manager Secret Key Creation, Updation and Deletion using Amazon CloudWatch event and Amazon SNS
+
+Learn more about this pattern at Serverless Land Patterns: << Add the live URL here >>
```

Comment:
add the url to the pattern - https://serverlessland.com/patterns/<folder for the pattern>